packaging/debian/control

```diff
@@ -0,0 +1,18 @@
+Source: azure-nvme-utils
+Section: utils
+Priority: optional
+Maintainer: Chris Patterson <cpatterson@microsoft.com>
+Build-Depends: cmake, pandoc, debhelper-compat (= 12)
+Standards-Version: 4.5.0
+Homepage: https://github.com/Azure/azure-nvme-utils
+Rules-Requires-Root: no
+#Vcs-Git: https://salsa.debian.org/debian/azure-nvme.git
+#Vcs-Browser: https://salsa.debian.org/debian/azure-nvme
```

Comment:
nit: either delete these or uncomment them and update them to point to the github repos

Reply:
done, thanks!